src/solve.jl

```diff
@@ -63,6 +63,7 @@ function DiffEqBase.__init(prob::Union{DiffEqBase.AbstractODEProblem,
     progress_steps = 1000,
     progress_name = "ODE",
     progress_message = ODE_DEFAULT_PROG_MESSAGE,
+    progress_id = gensym("OrdinaryDiffEq"),
```

Comment:
This needs to be added as a common interface arg in DiffEqBase and the docs

Reply:
I've added it in https://github.com/SciML/DiffEqBase.jl/pull/939 but not to the docs

Reply:
Ok docs updated as well